docs/DeveloperGuide.md

```diff
@@ -195,6 +210,36 @@ The following sequence diagram shows how a list is autosorted upon an addition o
     * Pros: Relatively low time complexity i.e. O(logn).
     * Cons: Prone to error and difficult to implement.
 
+### Done feature
+
+#### Implementation
+
+This Done feature allows users to mark their deliverables as completed. The user input is received by`MainWindow` in
+the `UI` component before being passed to `DeliverableLogicManager` to be executed. The `DeliverableLogicManager` will call
+`DeliverableBookParser` which will parse the command keyword ("done") to return a `DoneCommandParser`. `DoneCommandParser` will
+then parse the command argument to return a `DoneCommand`, which on execution will set the status of the specified deliverable
+to completed and update the `ModelDeliverable` accordingly. Invalid user inputs such as an invalid index will result in the
+appropriate error messages displayed to the user.
+
+Given below is a sequence diagram to show how the execution at each step.
```

Comment:
Should we use an ordered list here? Since they are described as 'steps' too

Reply:
I think we need to standardise cos chrystal also used prose